docs/Reference/quorum.js/RawTransactionManager.md

```diff
@@ -168,33 +168,46 @@ A promise that resolves to the transaction receipt if the transaction was sent s
 
 ## Examples
 
-### Externally signing and sending a private tx
+### Externally signing and sending a private transaction
+
+!!! info
 
-!!!info
     This is not supported by Constellation and requires Quorum v2.2.0+
 
-[Code sample](https://github.com/ConsenSys/quorum.js/blob/master/7nodes-test/deployContractViaHttp-externalSigningTemplate.js).
+The following workflow demonstrates how to externally sign and send a private transaction.
+See this
+[code sample](https://github.com/ConsenSys/quorum.js/blob/master/7nodes-test/deployContractViaHttp-externalSigningTemplate.js)
+as an example.
 
-1. `storeRawRequest` to encrypt the transaction `data`
+1. Use `storeRawRequest` to encrypt the transaction `data`.
 
     ```js
     txnManager.storeRawRequest(data, from)
     ```
 
-1. Replace `data` field of transaction with `key` field from `storeRawRequest` response
-1. Sign the transaction
-1. Mark the signed transaction as private with `setPrivate`
+2. Replace the `data` field of the transaction with the `key` field from the `storeRawRequest` response.
```

Comment:
use `1.` for all numbered lists items, ordered numbers will be generated automatically, do not define each number by yourself.